### PR TITLE
feat: DirectionProvider attaches dir attr, refactored Tooltip for RTL support, RTL mode switcher for docs

### PR DESCRIPTION
**Pull request by dozortsev**
## 🖼 Context

Helping @nabaroa with some RTL stuff https://github.com/transferwise/neptune-web/pull/1053.

## 🚀 Changes

- Added RTL switcher for docs https://transferwise.github.io/neptune-web/branch/dir-provider/about/Home which preserves mode across pages (based on `localStorage`)
- Refactored `Tooltip` component to use react-popper which enables RTL support
- Refactored `DirectionProvider` on TS and now it attaches `dir` attr on `html` as well
- Removed `storybook-addon-rtl` in favor of global know

## ✅ Checklist

- [ ] Make PR title meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint) (it will appear in the changelog)
- [ ] Changes are tested and all tests pass
- [ ] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/main/ACCESSIBILITY.md) and there are no violations in the console
- [ ] Changes work in all supported browsers (don't forget IE11)
- [ ] You've updated the documentation if necessary

### CODE REVIEW

File: packages/components/src/provider/direction/DirectionProvider.js [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
 
-export const DirectionProvider = ({ locale, children }) => (
-  <DirectionContext.Provider value={SUPPORTED_RTL.includes(locale) ? 'rtl' : 'ltr'}>
-    {children}
-  </DirectionContext.Provider>
-);
+export const DirectionProvider = ({ direction, children }) => {
+  return (
+    <DirectionContext.Provider value={direction}>
+      <div dir={direction}>{children}</div>
```
Comment (nabaroa) [inline packages/components/src/provider/direction/DirectionProvider.js:9]: Can we add the `dir` attribute to the HTML element here?

Comment (dozortsev) [reply packages/components/src/provider/direction/DirectionProvider.js]: done

File: packages/components/.storybook/main.js [hunk 2]
```diff
@@ -15,7 +15,6 @@ module.exports = {
     '@storybook/addon-a11y',
     '@storybook/addon-docs',
     '@storybook/addon-viewport',
-    'storybook-addon-rtl',
```
Comment (dozortsev) [inline packages/components/.storybook/main.js:18]: Storybook doesn’t remember addon configuration after interaction with component which is really annoying and inconvenient


https://user-images.githubusercontent.com/4015628/132008029-a2467d51-475b-4c20-862a-be3267e35b7e.mov

instead we suggesting to use global knob


https://user-images.githubusercontent.com/4015628/132008144-3cba70e4-5a8a-467a-a268-d55e4db5d0ce.mov

File: packages/components/.storybook/preview.js [hunk 3]
```diff
@@ -93,21 +88,20 @@ const severalExamplesOfSupportedLocales = [
  */
 const VisualTestDecorator = (storyFn) => {
   const urlParams = new URLSearchParams(document.location.search.substring(1));
-
-  // enable RTL mode
-  document.documentElement.dir = urlParams.get('direction') || 'ltr';
-  return <>{storyFn()}</>;
+  const direction = urlParams.get('direction') || 'ltr';
+  return <DirectionProvider direction={direction}>{storyFn()}</DirectionProvider>;
```
Comment (dozortsev) [inline packages/components/.storybook/preview.js:92]: replacing this logic with DirectionProvider as it does exactly same

File: packages/docs/components/Layout.js [hunk 4]
```diff
@@ -19,6 +21,19 @@ const Layout = ({ children, router: { pathname } }) => {
   const page = getPageFromPath(pathname);
   const editPath = `${githubURL}${pathname}.mdx`;
 
+  let directionDefaultState = false;
+
+  if (isBrowser()) {
+    directionDefaultState = JSON.parse(localStorage.getItem('isRTL')) || false;
```
Comment (matebek-tw) [inline packages/docs/components/Layout.js:27]: This would be more meaningful to call `isRightToLeft`.

**Review by matebek-tw — APPROVED** [review]

**Pull request merged**